build/windows/installer/conf/fluent-bit-common.conf

```diff
@@ -47,6 +47,7 @@
     TelemetryPushIntervalSeconds    300
     Match                           oms.container.*
 
+# TODO: Merge this with oms_output plugin as both of them are using the same plugin
```

Comment:
are you plan to merge with oms_output plugin?

Reply:
will run some tests again and decide